### DISCUSSION TITLE
Better ship progression

### DISCUSSION THREAD

**Discussion by AircoDHK** [endless-sky/endless-sky#8657] [Ideas]
**First of all I'm not really familiar with the way Github works or how spoilers are handled so I just want to say that I will be talking about content available later on in the game and that if you want to discover Endless sky on your own, which i recommend, you should not read this.**

In this thread I want to talk about the abrupt difference between mid game ships and late game ships I experienced and my view on how to solve this issue.

I currently did not finish the FW campaign because I cant really find time to do it, however I have a fleet consisting of 3 geocoris, 10 shield beetles and 6 sea scorpions, each of these ships are outfitted to my best. My flag ship is a korath rano'erek.

My main point is that kor mereti and kor sestor ships are not a viable milestone between shield beetles and tier 2.5-3 ships in their current state in my opinion but i will get to it later.

After a look at the stats of ships, I realised that shield beetles have a lot of health for a 1.5 tier ship ( nearly twice as much that of a falcon), still not even close to kor mereti ships , which are only tier 2. So I understand that the tier system is not perfect because it describes a faction as a whole, but my point still stands because i can't find any ships other than kor mereti/sestor between shield beetles and heliachs ships that is agressive towards the player so I don't ruin my storyline with the wanderers.

Now the reason I think kor sestor/mereti farming is not enjoyable currently is because 

1. The system is completely based on RNG 
2. Even without RNG actively farming them is a pain because of the sheer number of them that keeps jumping to the system you're in.
3. I've encountered several persons on forums saying you can farm them passively with a remnant ship with cloak. Which I tried without any results, so I may be doing it wrong somehow but still being afk for three hours just to have a model 64 when they are equivalent to shield beetles does not sound fun to me.

That's my last complain and now I want to talk about the ideas I tought could be good to make this step in the game more fun.

I complained about the RNG system, however I understand you can't just let a player capture a tier 2 ship with a shuttle because it has no crew on it, which is even more op after you capture it cause you don't pay. That's why I think there should be a new outfit called something like "hacking device" that is only sold in korath effreti outfitters after you complete a chain of quests that need shield beetles or korath exiles ships to complete. This outfit, when equipped on your ship will reduce the odds of capturing a ship maybe to 40% or along the lines of this depending on the ship( You currently have only 4% chance of capturing a model 512). This outfit could take 15 outfit space and only used once so you can't farm them on a too large scale. This would prevent players from having an army of automated ships too early while still giving the possibility of having these ships to players who want to actively fight them and capture them.

Another idea I had but I think will be more difficult to implement would be to have some kind of minigame that pops up when you try to hack a ship. Completing the minigame would be needed to capture of the ship and maybe time could flow normally when doing the minigame so you can't just farm them with your remnant ship. This of course could be implemented on par with the hacking device so that the minigame only pops up if you have one installed. Also I get it, a minigame to hack something in a videogame is not very original and doesn't really fit in the game's style so if a developer read this may they at least remember the time flowing when capturing a ship, because it would make capturing ships if you don't have escorts to protect you much less op.

Sorry if I made a thread this long for only two ideas, I really needed to share my frustration regarding the capture of kor ships, and if you made it till here, first thank you and I hope I could discuss this with other people.

**Comment by mOctave** [comment]
> That's why I think there should be a new outfit called something like "hacking device" that is only sold in korath effreti outfitters after you complete a chain of quests that need shield beetles or korath exiles ships to complete.

See https://github.com/endless-sky/endless-sky/pull/8506

_Edit: Oops, it looks like this wasn't quite what you were talking about. Nonetheless, it's a similar feature that could help slightly in this area._

**Reply by AircoDHK** [reply]
Thanks for the reply, and after reading this idea it is undeniably more complete and in depth than the thing i posted. My hacking device only changes the chances of capturing the ship, which is a bit archaique. So yeah this is the kind of outfit I was thinking about however I still think capturing ships while time is passing by would balance the current meta where you rush a ship with a lot of crew and then capture every ship you encounter, so you actually need your escorts to shield you while you capture.
